### DISCUSSION TITLE
There is no way 1.6 to 1.7 and then 8 ??

### DISCUSSION THREAD

**Discussion by mertyakan** [PrestaShop/PrestaShop#34183] [Q&A] [answered]
Hello,

I tried upgrading my Prestashop website. I'm using 1.6.1.24 now. I looked at the Github issues and auto-upgrade documentation. I found that it is not possible to migrate from the current version to the latest version. Once from 1.6 to the latest version 1.6 and from 1.6 to 1.7. And then 8.

I used 1.6.1.24 successfully... I used One Click Upgrade.

And then.. I tried the latest 1.7 (PrestaShop 1.7.8.10 https://github.com/PrestaShop/PrestaShop/releases/download/1.7.8.10/prestashop_1.7.8.10.zip)

Stopped during upgrade and asked to restore. Presta is broken. I restored the file and database backup. I tried different versions (1.7.8.2 1.7.0.1) and still got the same error. Maybe it will be funny but I tried it from the latest version 1.6 to 8 😆

I'm re-searching for execution or another log. There is nothing.

Is there really no way from 1.6 to 1.7?

**Comment by matks** [comment, marked as answer]
Hello @mertyakan there is no direct way to upgrade from 1.6 to 8 because there are too many major changes between 1.6, 1.7 and 8

But what you can do is
1. Use AutoUpgrade [v4.14.3](https://github.com/PrestaShop/autoupgrade/releases/tag/v4.14.3) to upgrade from 1.6 to 1.7.
2. When your shop is on 1.7, use latest AutoUpgrade module to upgrade from 1.7 to 8

When doing step 1 make sure to use the right channel https://devdocs.prestashop-project.org/8/basics/keeping-up-to-date/upgrade-module/channels

However this is just for how to upgrade the Core. Please be aware it is very unlikely that the modules you have enabled on your 1.6 shop will work on your v8 shop. I would suggest to that 1st you disable all of them, then you perform your shop upgrade, then you re-enable them one by one. There might be for some/all of these modules a newer version that is compatible with v8: then you'll need to upgrade the modules too.

**Comment by mertyakan** [comment]
Hi matks, 

I used AutoUpgrade v4.14.3 (https://github.com/PrestaShop/autoupgrade/releases/tag/v4.14.3) to upgrade.

But by default minor channel version: Latest official version for minor channel.: 1.6.1 stable - (1.6.1.24)

When I converted it to the mayor channel: Latest official version for major channel.: 8.1 stable - (8.1.2)

I didn't see any channel to upgrade to 1.7. That's why I tested to 1.7 as follows.

![A](https://mertyakan.com/wp-content/uploads/2023/10/Screenshot-2023-10-05-at-23.39.16.png)

**Reply by mertyakan** [reply]
By the way, I don't use any special modules. No theme is installed. Everything is default. There are no plugins. Simple prestashop.

**Reply by matks** [reply]
Apologies I did a wrong copy-paste in my 1st message (now fixed): there is more than Minor or Major channel 😉 click on "Expert mode" and use Private or Archive channel https://devdocs.prestashop-project.org/8/basics/keeping-up-to-date/upgrade-module/channels/#major

You'll have to download the 1.7 ZIP you want to upgrade to and give it manually to Upgrade assistant
